tutorials/Tutorial2_Finetune_a_model_on_your_data.ipynb

```diff
@@ -0,0 +1,299 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fine-tuning a model on your own data\n",
+    "\n",
+    "This tutorial shows you how to fine-tune a pretrained model on your own dataset for the task of question-answering."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from haystack import Finder\n",
+    "from haystack.database.sql import SQLDocumentStore\n",
+    "from haystack.indexing.cleaning import clean_wiki_text\n",
+    "from haystack.indexing.io import write_documents_to_db, fetch_archive_from_http\n",
+    "from haystack.reader.farm import FARMReader\n",
+    "from haystack.retriever.tfidf import TfidfRetriever\n",
+    "from haystack.utils import print_answers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training\n",
+    "We take a reader as a base model and fine-tune it on our own custom dataset (should be in SQuAD-like format)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "03/17/2020 12:49:49 - INFO - farm.utils -   device: cpu n_gpu: 0, distributed training: False, automatic mixed precision training: None\n",
+      "03/17/2020 12:49:49 - INFO - farm.infer -   Could not find `distilbert-base-uncased-distilled-squad` locally. Try to download from model hub ...\n",
+      "03/17/2020 12:49:54 - WARNING - farm.modeling.language_model -   Could not automatically detect from language model name what language it is. \n",
+      "\t We guess it's an *ENGLISH* model ... \n",
+      "\t If not: Init the language model by supplying the 'language' param.\n",
+      "03/17/2020 12:49:57 - WARNING - farm.modeling.prediction_head -   Some unused parameters are passed to the QuestionAnsweringHead. Might not be a problem. Params: {\"loss_ignore_index\": -1}\n",
+      "03/17/2020 12:49:59 - INFO - farm.utils -   device: cpu n_gpu: 0, distributed training: False, automatic mixed precision training: None\n",
+      "03/17/2020 12:49:59 - INFO - farm.utils -   device: cpu n_gpu: 0, distributed training: False, automatic mixed precision training: None\n",
+      "Preprocessing Dataset squad20/dev-v2.0.json: 100%|██████████| 1204/1204 [00:19<00:00, 62.20 Dicts/s]\n",
+      "Train epoch 1/1 (Cur. train loss: 4.7116):   0%|          | 5/1193 [00:56<3:40:15, 11.12s/it]\n\n"
+     ]
+    }
+   ],
+   "source": [
+    "reader = FARMReader(model_name_or_path=\"distilbert-base-uncased-distilled-squad\", use_gpu=False)\n",
+    "train_data = \"squad20\"\n",
```

Comment:
Maybe use "data/squad20"? So it checks this folder, if it doesnt exists or is empty it will download the files there.

Reply:
Please also adjust this in colab.